yb-voyager/cmd/importSchema.go

```diff
@@ -90,23 +90,40 @@ func importSchema() {
 	// Import ALTER TABLE statements from sequence.sql only after importing everything else
 	isAlterStatement := func(objType, stmt string) bool {
```

Comment:
I think we can change this `isAlterStatement` to `isSkipStatement` to make it general. Will do.

Reply:
or we can break it into multiple specific ones like `isALTERStatemet` , `createUniqueIndex` etc...

Reply:
That is something I was also thinking but then as we pass only one skipFn to importSchemaInternal, we have to pass multiple functions to it I guess.

Reply:
yeah, ok, maybe you can multiple conditions named separately and then add and pass them in one skipFn

Reply:
Yes, that can be done.